src/Sylius/Bundle/MoneyBundle/Formatter/AmountFormatter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\MoneyBundle\Formatter;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+final class AmountFormatter implements AmountFormatterInterface
+{
+    /**
+     * @param int $amount
+     * @param string $currency
+     * @param bool $decimal
+     * @param string $locale
+     *
+     * @return string
+     */
+    public function format($amount, $currency, $decimal = false, $locale = 'en')
```

Comment:
IMO `$decimal` is redundant here - if currency is given, use `\NumberFormatter::CURRENCY`, if not  `\NumberFormatter::DECIMAL`. If we require currency to be passed, we can just delete that option, because currency won't be used anyway.